lib/query-types.d.ts

```diff
@@ -1,16 +1,15 @@
-
-export = {
-  SELECT: 'SELECT',
-  INSERT: 'INSERT',
-  UPDATE: 'UPDATE',
-  BULKUPDATE: 'BULKUPDATE',
-  BULKDELETE: 'BULKDELETE',
-  DELETE: 'DELETE',
-  UPSERT: 'UPSERT',
-  VERSION: 'VERSION',
-  SHOWTABLES: 'SHOWTABLES',
-  SHOWINDEXES: 'SHOWINDEXES',
-  DESCRIBE: 'DESCRIBE',
-  RAW: 'RAW',
-  FOREIGNKEYS: 'FOREIGNKEYS'
-};
+export enum QueryTypes {
```

Comment:
this is not how it is in the source file.
Please add back `export = QueryTypes` at the bottom.

Reply:
As described in my comment here: https://github.com/types/sequelize/pull/141#discussion_r147594431 using `export = QueryTypes` causes a compiler error.

It seems like the only other option is to declare `QueryTypes` globally using `declare enum QueryTypes` as done in [this commit](https://github.com/types/sequelize/pull/141/commits/79347c22371349a717d1d2d055a736173be71431).

I am fairly new to Typescript, so perhaps I'm missing something. Why are we trying to mirror the export style in the Sequelize library?